signer/src/transaction_signer.rs

```diff
@@ -500,23 +501,36 @@ where
     }
 
     /// Process WSTS messages
-    #[tracing::instrument(skip_all, fields(txid = %msg.txid))]
+    #[tracing::instrument(skip_all, fields(
+        msg_id = %msg.id,
+        msg_type = %msg.type_id(),
+        dkg_signer_id = tracing::field::Empty,
+        dkg_id = tracing::field::Empty,
+        dkg_sign_id = tracing::field::Empty,
```

Comment:
Why call these fields `dkg_sign_id` and `dkg_iter_id`?  They aren't present in DKG messages.  Same with `dkg_signer_id`, though `signer_id` at least is present in many DKG messages but also elsewhere.

Reply:
Good point, will revisit

Reply:
Updated in [8cdb739](https://github.com/stacks-network/sbtc/pull/1287/commits/8cdb739276f631677f6a1ae4d8b1604ace59a5cd).

I'm curious if `dkg_id` and `sign_id` are conditionally set depending if it's a dkg round vs. signing? If so then if the default is `0` then we could `record()` them only if `> 0` so that only the relevant one is logged.

Reply:
`dkg_id` is always present, in dkg or signing rounds.  `sign_id` is only present in signing rounds.  

they are always set if they are present.